RNA_pipeline/ExonUsage_hg38_fixed.wdl

```diff
@@ -0,0 +1,31 @@
+task ExonUsage {
+  String sample_id
```

Comment:
do we ever use this wdl?

Reply:
No but my tool export to everything that is in the workspace. And it is in the workspace. we can remove if we want to

Reply:
gotcha. i'm fine with keeping it